### ISSUE TITLE
TensorFlow Slim: Inception V2 Confusion

### ISSUE THREAD

**Issue by dichotomies** [tensorflow/models#5054]
On https://github.com/tensorflow/models/tree/master/research/slim, `Inception V2` refers to [Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](https://arxiv.org/abs/1502.03167).

This reference seems contrary to the definition of Inception V2 in the paper [Rethinking the Inception Architecture for Computer Vision](https://arxiv.org/pdf/1512.00567.pdf), because it is introduced there.

The paper you refer to is about BN-Inception if I'm not mistaken.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by karmel** [comment]
The Slim Inception model is no longer actively maintained. That said, if you submit a PR, I can find someone with expertise to review.

**Comment by dichotomies** [comment]
I've submitted a PR: https://github.com/tensorflow/models/pull/5151.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
